Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -9246,6 +9246,111 @@ def get_template_stacks(
     return PanoramaCommand.get_template_stacks(topology, device_filter_string)
 
 
+def get_global_counters(topology: Topology, device_filter_string: str = None) -> ShowCounterGlobalCommmandResult:
```

Comment:
```suggestion
def get_global_counters(topology: Topology, device_filter_string: Optional[str] = None) -> ShowCounterGlobalCommmandResult:
```

Change it to all the functions below as well (if necessary).
Whenever you have a default value of ```None``` make sure its wrapped with Optional which basically means in that specific case that device_filter_string could be None or a string